pyproject.toml

```
@@ -32,5 +32,6 @@ markers = [
     "slowtest",
     "link_check",
     "parametrize_any: only execute one of the parameterized cases when in fast mode (see documentation in conftest.py)",
-    "db_restore_dump(dump): mark the db dump to restore. To be used in conjunction with the `migrate_db_from` fixture."
+    "db_restore_dump(dump): mark the db dump to restore. To be used in conjunction with the `migrate_db_from` fixture.",
+    "migration_test: mark a test as a migration test to skip it in fast runs once it is older than 30 days"
```

Let's make it very clear that we are talking about a *database* migration script. It's likely that we will have other types of migrations in the future.

```suggestion
    "db_migration_test: mark a test as a migration test to skip it in fast runs once it is older than 30 days"
```